packages/toolkit/src/query/react/buildHooks.ts

```diff
@@ -465,7 +465,7 @@ export function buildHooks<Definitions extends EndpointDefinitions>({
   api,
   moduleOptions: { batch, useDispatch, useSelector, useStore },
 }: {
-  api: Api<any, Definitions, any, any, CoreModule>
+  api: Api<any, Definitions, any, any>
```

Comment:
this seems to be a type change. was that intended?

Reply:
Ah, this change coming from WebStorm IDE support that says

"_Explicit type argument any, Definitions, any, any, CoreModule can be replaced with any, Definitions, any, any_"
<img width="1173" alt="Screen Shot 2021-09-04 at 23 03 49" src="https://user-images.githubusercontent.com/5501268/132097753-83941e25-99ef-4c2c-8ac6-c9bf066ba0bc.png">

Because type `API`'s 4th generics argument has set default type `CoreModule`.

<img width="551" alt="Screen Shot 2021-09-04 at 23 04 33" src="https://user-images.githubusercontent.com/5501268/132098157-157a58c6-f17f-42fd-9aab-9c32bded899b.png">

Would you prefer bring back original explicit way?  
Personally original is more friendly for external contributors.